src/shogun/statistics/QuadraticTimeMMD.cpp

```diff
@@ -1022,8 +1024,9 @@ SGVector<float64_t> CQuadraticTimeMMD::sample_null_spectrum_DEPRECATED(
 	}
 
 	/* when m=n, return m*MMD^2 instead */
-	if (m==n)
-		null_samples.scale(0.5);
+	if (m==n) {
+		linalg::scale<linalg::Backend::NATIVE>(null_samples, 0.5);
```

Comment:
Yes, now I see that I've missed braces style when reading README_developer. Thanks for comment!